src/python/doc/datasets_generators.inc

```diff
@@ -0,0 +1,14 @@
+.. table::
+   :widths: 30 40 30
+
+   +-----------------------------------+--------------------------------------------+--------------------------------------------------------------------------------------+
+   | | :math:`(x_1, x_2, \ldots, x_d)` | Datasets generators (points).              | :Authors: Hind Montassif                                                             |
```

Comment:
It would be prettier to use an image of points on a sphere or a torus.

Reply:
Yes, I am not sure the one I used is okay (sphere generated with plot_trisurf and 10 000 n_samples).

Reply:
Changed it for another one using scatter instead of trisurf.